charts/puris-backend/values.yaml

```diff
@@ -0,0 +1,129 @@
+# Default values for puris-backend.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
```

Comment:
Does this `values.yaml` reflect all configuration settings from `src/main/resources/application.properties` mentioned in `INSTALL.md`. In the wild one will use the helm chart only to install, and won't build a custom image etc.

Reply:
Yes, it does, since everything mandatory is filled with default values.

To be really sure, I checked out the repository on a new machine and tried to deploy the application via helm without any configuration, and it worked like a charm!

Reply:
OK, but I didn't get it yet. When PURIS needs connection to a database and an EDC (according to `applications.properties`) where to configure the EDC and DB connection in the Chart? Right now it seems to be hardcoded in the image?!